packages/webview_flutter/webview_flutter_android/lib/src/android_webview_api_impls.dart

```diff
@@ -94,6 +99,7 @@ class AndroidWebViewFlutterApis {
       WebChromeClientFlutterApi.setup(webChromeClientFlutterApi);
       JavaScriptChannelFlutterApi.setup(javaScriptChannelFlutterApi);
       FileChooserParamsFlutterApi.setup(fileChooserParamsFlutterApi);
+      WebViewFlutterApi.setup(webViewFlutterApi);
```

Comment:
Thanks for adding this! I hadn't realized this was missing when I added the `WebViewFlutterApi`.